integrations/spark/spark-job-dataquality/spark-job-dataquality-app/src/main/java/com/thinkbiganalytics/spark/dataquality/output/DataQualityRow.java

```diff
@@ -0,0 +1,169 @@
+package com.thinkbiganalytics.spark.dataquality.output;
+
+/*-
+ * #%L
+ * thinkbig-spark-job-dataquality-app
+ * %%
+ * Copyright (C) 2017 ThinkBig Analytics
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+
+import java.io.Serializable;
+
+import com.thinkbiganalytics.spark.dataquality.rule.DataQualityRule;
+
+/**
+ * Class to represent a row in data quality output<br>
+ * Format of output:<br>
+ *
+ * RuleName, Description, Status, Comments
+ */
+@SuppressWarnings("serial")
+public class DataQualityRow implements Serializable {
+
+    /**
+     * Need to suppress squid:S00116 because these variables
+     * names need to align with Hive table column names
```

Comment:
@azfaraziz It looks like the temp table is using camel-case instead of underscores, and the output table is only indirectly related since it's doing an insert ... select. Could you clarify why these variable names need to align with the Hive table column names?